api/client/builder/bid.go

```diff
@@ -156,7 +159,7 @@ func WrappedBuilderBidCapella(p *ethpb.BuilderBidCapella) (Bid, error) {
 
 // Header --
 func (b builderBidCapella) Header() (interfaces.ExecutionData, error) {
-	return blocks.WrappedExecutionPayloadHeaderCapella(b.p.Header)
+	return blocks.WrappedExecutionPayloadHeaderCapella(b.p.Header, big.NewInt(0).SetBytes(bytesutil.ReverseByteOrder(b.p.Value)))
```

Comment:
When I look at this line in a vacuum, there exists a hidden coupling because you're tied to expecting a big endian ordering from b.p.Value. What if this changes in the future? then you'll be reversing into an ordering you do not want. Instead, I would be explicit here about the byte ordering you want

Reply:
If this change in the future then it'll be a dramatic hard fork change then we'll have to sweep the entire client codebase for all the instances. Things will fail abruptly and I argue that's ok. This is the biggest technique ethereum debt where we chose little endian for consensus layer